ister_gui.py

```diff
@@ -2395,10 +2474,18 @@ def _save_config(self, config, mount_d):
                     output = ''
                 if 'TYPE="' in output:
                     config['FilesystemTypes'][-1]['disable_format'] = True
+            if "encryption" in mount_d[point]:
+                config['FilesystemTypes'][-1]["encryption"] = {
+                    "passphrase": mount_d[point]['encryption'],
+                    "name" : "encrypted_dev" }
             config['PartitionMountPoints'].append({
                 'disk': disk,
                 'partition': part,
                 'mount': point})
+            if "encryption" in mount_d[point]:
+                config['PartitionMountPoints'][-1]["encryption"] = {
+                    "name" : "encrypted_dev" }
+                config["Bundles"].append("bootloader-extras")
```

Comment:
This needs to be called out in the bundle screen if we are adding an additional bundle. We can't just silently add bundles, even if it is a relatively small bundle (about 3.6 MB).

Reply:
I will do

Reply:
It should be added to the required bundles list if this option is set, and the optional bundles list if not - or maybe absent entirely. Additionally there needs to be some messaging on **this** screen to tell the user encryption requires the bootloader-extras bundle.

Reply:
of course I already seen that :)

Reply:
done